openapi/openapi.yml

```diff
@@ -899,6 +899,33 @@ paths:
           $ref: '#/components/responses/Error'
       tags:
         - Postgres Metric Destination
+  '/project/{project_id}/location/{location}/postgres/{postgres_database_name}/read-replica':
```

Comment:
We replaces `postgres_database_name` with `postgres_database_reference` in this [PR](https://github.com/ubicloud/ubicloud/pull/3150)